itest/lnd_route_blinding_test.go

```diff
@@ -349,11 +349,24 @@ func newBlindedForwardTest(ht *lntest.HarnessTest) (context.Context,
 func (b *blindedForwardTest) setupNetwork(ctx context.Context,
 	withInterceptor bool) {
 
+	const chanAmt = btcutil.Amount(100000)
```

Comment:
nit: I always like the `100_000` notation.

Reply:
cool changed